lib/Doctrine/ORM/Mapping/Driver/AttributeDriver.php

```diff
@@ -397,12 +417,66 @@ public function loadMetadataForClass($className, ClassMetadata $metadata): void
             }
         }
 
+        // Evaluate AssociationOverrides annotation
+        if (isset($classAttributes[Mapping\AssociationOverrides::class])) {
+            $associationOverride = $classAttributes[Mapping\AssociationOverrides::class];
+
+            foreach ($associationOverride->overrides as $associationOverride) {
+                $override  = [];
+                $fieldName = $associationOverride->name;
+
+                // Check for JoinColumn/JoinColumns annotations
```

Comment:
`#[JoinTable(inverseJoinColumns: [], joinColumns: [])]`

Here, there is no annotation, and `JoinTable` is an attribute. How should we name `inverseJoinColumns` and `joinColumns`? The docs seem to be referring to those as … attributes: https://www.doctrine-project.org/projects/doctrine-orm/en/2.10/reference/attributes-reference.html#column

Should we use "parameters" here instead?

There is the same issue in other comments below.